modules/auth0/tenant/main.tf

```diff
@@ -0,0 +1,89 @@
+locals {
+  enabled = module.this.enabled
+
+  name        = length(module.this.name) > 0 ? module.this.name : "auth0"
+  domain_name = format("%s.%s", local.name, module.dns_gbl_delegated.outputs.default_domain_name)
+
+  friendly_name = length(var.friendly_name) > 0 ? var.friendly_name : module.this.id
+}
+
+# Chicken before the egg...
+#
+# The tenant must exist before we can manage Auth0 with Terraform,
+# but the tenant is not a resource identifiable by an ID within the Auth0 Management API!
+#
+# However, we can import it using a random string. On first run, we import the existing tenant
+# using a random string. It does not matter what this value is. Terraform will use the same
+# tenant as the Auth0 application for the Terraform Auth0 Provider.
+#
+# https://registry.terraform.io/providers/auth0/auth0/latest/docs/resources/tenant#import
+import {
+  to = auth0_tenant.this[0]
+  id = "f6615002-81ff-49f7-afd3-8814d07af4fa"
+}
```

Comment:
I think this is a leftover from development work

Reply:
It's a workaround for a nonexistent identifier in Auth0

See [this comment above
](https://github.com/cloudposse/terraform-aws-components/blob/7aac04bb96d00d64c59e15410e3e52c2d975c24e/modules/auth0/tenant/main.tf#L10-L19)

And the README: https://github.com/cloudposse/terraform-aws-components/blob/7aac04bb96d00d64c59e15410e3e52c2d975c24e/modules/auth0/tenant/README.md#auth0-tenant-creation

Reply:
Ah got it. Missed that.